doc/nrf/ug_thread_commissioning_configuring.rst

```diff
@@ -0,0 +1,175 @@
+﻿.. _thread_ot_commissioning_configuring_on-mesh:
+
+Configuring on-mesh Thread commissioning with CLI sample
+########################################################
+
+You can configure on-mesh Thread commissioning using the :ref:`Thread CLI sample <ot_cli_sample>` with two devices to form a Thread network.
+One device will act as a Commissioner and the other will be a Joiner.
+
+.. note::
+    Before you start the configuration process, make sure you are familiar with :ref:`Thread commissioning concepts <thread_ot_commissioning>`, especially :ref:`thread_ot_commissioning_types_on-mesh`.
+
+.. _thread_ot_commissioning_configuring_on-mesh_cli_requirements:
+
+Requirements
+============
+
+* Two of the following development kits:
+
+  * |nRF52840DK|
+  * |nRF52833DK|
+
+.. _thread_ot_commissioning_configuring_on-mesh_cli_flashing:
+
+Flashing the boards
+===================
+
+Flash both development kits with the :ref:`Thread CLI sample <ot_cli_sample>`.
+See the sample's page for details.
+
+After flashing, both devices are pre-commissioned to the same Thread network.
+This means that if ``channel`` and ``panid`` are set to the same value on both devices, they both become members of the same Thread network after entering the following commands:
```

Comment:
The NCS CLI example comes with the autostart feature, so the devices will form the network automatically without user intervention. In this section we should indicate how to disconnect from the network prior to start with the commissioning:

`ot thread stop`